modules/contacts/contacts.resolver.js

```diff
@@ -0,0 +1,43 @@
+const contactsService = require('./contacts.service');
+
+const contactsQuery = {
+  getContacts: () => contactsService.getContacts(),
+  getContactById: (_, args) => contactsService.getContactById(args.id),
```

Comment:
it is better to use "parent" instead __. Please change getContactById: ( _ , args) =... to getContactById: ( parent , args) =...